src/tracing/trace.test.ts

```diff
@@ -25,13 +25,11 @@ function getTransaction(index = 0): Transaction {
 
 describe('trace', () => {
   beforeEach(() => {
-    mocked(Sentry.startTransaction)
-      .mockReset()
-      .mockImplementation((context) => {
-        const transaction: Transaction = jest.requireActual('@sentry/react').startTransaction(context)
-        transaction.initSpanRecorder()
-        return transaction
-      })
+    mocked(Sentry.startTransaction).mockImplementation((context) => {
```

Comment:
This change looks like something that could be a good candidate for a follow-up PR, as it seems to be unrelated?

Reply:
I've included removing mock resets in this PR.